external/openmeeg/om_save_tri.m

```diff
@@ -8,6 +8,14 @@
 
 % Copyright (C) 2010-2017, OpenMEEG developers
 
+% OpenMEEG vs 2.3 and up internally adjusts the convention for surface
+% normals, but OpenMEEG v2.2 expects surface normals to point inwards;
+% this checks and corrects if needed
+ok = checknormals(points,faces);
+if ~ok
+    faces = fliplr(faces);
+end
```

Comment:
can you do this optionally ie add a check_normals parameter to the function that fieldtrip sets to true ? Some people maybe just want to read a tri and write back a tri file to disk, and they would not expect to have their normals automatically flipped.